README-hi.md

```diff
@@ -0,0 +1,82 @@
+
+* [योगदान देना शुरू करें](https://kubernetes.io/docs/contribute/start/)
+* [Staging Your Changes](http://kubernetes.io/docs/contribute/intermediate#view-your-changes-locally)
+* [पेज टेम्पलेट](http://kubernetes.io/docs/contribute/style/page-templates/)
+* [प्रलेखन शैली गाइड](http://kubernetes.io/docs/contribute/style/style-guide/)
+* [स्थानीयकरण कुबरनेट्स प्रलेखन](https://kubernetes.io/docs/contribute/localization/)
+
+## `README.md`'s स्थानीयकरण कुबरनेट्स प्रलेखन
+
+आप पर हिंदी स्थानीयकरण के maintainers तक पहुँच सकते हैं:
+
+* Yashu Mittal ([Twitter](https://twitter.com/mittalyashu77), [GitHub](https://github.com/mittalyashu))
+* [Slack channel](https://kubernetes.slack.com/messages/kubernetes-docs-hi)
+
+## स्थानीय रूप से Docker का उपयोग करके साइट चलाना
+
+कुबरनेट्स वेबसाइट को स्थानीय रूप से चलाने के लिए अनुशंसित तरीका एक विशेष [Docker](https://docker.com) image को चलाना है, जिसमें [Hugo](https://gohugo.io) स्टेटिक साइट जनरेटर शामिल है।
+
+> यदि आप विंडोज पर हैं, तो आपको कुछ और टूल्स की आवश्यकता होगी जिन्हें आप [Chocolatey](https://chocolatey.org) के साथ इंस्टॉल कर सकते हैं।
+
+> यदि आप Docker के बिना स्थानीय रूप से वेबसाइट चलाना पसंद करते हैं, तो नीचे Hugo का उपयोग करके स्थानीय रूप से साइट चलाना देखें।
+
+यदि आप Docker के बिना स्थानीय रूप से वेबसाइट चलाना पसंद करते हैं, तो नीचे दिए गए Hugo का उपयोग करके स्थानीय रूप से [साइट को चलाने](#running-the-site-locally-using-hugo) का तरीका देखें।
+
+यदि आपके [पास डोकर चल रहा है](https://www.docker.com/get-started), तो स्थानीय रूप से `कुबेरनेट्स-ह्यूगो` Docker image बनाएँ:
+
+```bash
+make docker-image
+```
+
+एक बार image बन जाने के बाद, आप साइट को स्थानीय रूप से चला सकते हैं:
+
+```bash
+make docker-serve
+```
+
+साइट देखने के लिए अपने browser को `http: // localhost: 1313` पर खोलें। जैसा कि आप source फ़ाइलों में परिवर्तन करते हैं, Hugo साइट को अपडेट करता है और browser को refresh करने पर मजबूर करता है।
+
+## Hugo का उपयोग करते हुए स्थानीय रूप से साइट चलाना
+
+Hugo निर्देशों के लिए [आधिकारिक हHugo प्रलेखन](https://gohugo.io/getting-started/installing/) देखें। [`Netlify.toml`](netlify.toml#L9) फ़ाइल में` HUGO_VERSION` environment variable द्वारा निर्दिष्ट Hugo संस्करण को install करना सुनिश्चित करें।
+
+जब आप Hugo को install करते हैं तो स्थानीय रूप से साइट को चलाने के लिए:
+
+```bash
+make serve
+```
+
+यह पोर्ट `1313` पर Hugo सर्वर को शुरू करेगा। साइट देखने के लिए अपने browser को `http://localhost:1313` पर खोलें। जैसा कि आप source फ़ाइलों में परिवर्तन करते हैं, Hugo साइट को अपडेट करता है और एक browser को refresh करने पर मजबूर करता है।
+
+## समुदाय, चर्चा, योगदान और समर्थन
+
+[Community page](http://kubernetes.io/community/) पर कुबरनेट्स समुदाय के साथ जुड़ना सीखें।
+
+आप इस परियोजना के स्थानीयकरण तक पहुँच सकते हैं:
+
+- [Slack](https://kubernetes.slack.com/messages/sig-docs)
+- [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-docs)
+
+## आचार संहिता
+
+कुबरनेट्स समुदाय में भागीदारी [कुबरनेट्स आचार संहिता द्वारा](code-of-conduct.md) शासित है.
```

Comment:
You can the link to the localized version of the Code of Conduct.

Reply:
Sure.

Reply:
I have updated the link to https://github.com/cncf/foundation/blob/master/code-of-conduct-languages/hi.md

This file might not be accessible not, once the [PR](https://github.com/cncf/foundation/pull/39) is merged it should work.